sensor/kubernetes/complianceoperator/updater.go

```diff
@@ -124,7 +124,7 @@ func (u *updaterImpl) collectInfoAndSendResponse() bool {
 
 func (u *updaterImpl) getComplianceOperatorInfo() *central.ComplianceOperatorInfo {
 	var err error
-	var ns string
+	ns := u.complianceOperatorNS
 	if u.complianceOperatorNS == "" {
```

Comment:
With above change this can be changed to
```suggestion
	if ns == "" {
```